CHANGELOG/CHANGELOG-3.6.md

```diff
@@ -8,7 +8,11 @@ Previous change logs can be found at [CHANGELOG-3.5](https://github.com/etcd-io/
 
 <hr>
 
-## v3.6.0-rc.1 (TBD)
+## v3.6.0-rc.2 (TBD)
```

Comment:
I've added this placeholder but we can just delete it if we decide to publish `v3.6.0` without an rc2 release.

Reply:
We need rc.2 because we need to backport https://github.com/etcd-io/etcd/pull/19452

Reply:
Let's target rc.2 next week.